webapp/src/components/table/table.test.tsx

```diff
@@ -383,7 +383,7 @@ describe('components/table/Table extended', () => {
     test('should match snapshot with UpdatedBy', async () => {
         const board = TestBlockFactory.createBoard()
 
-        const modifiedById = Utils.createGuid()
+        const modifiedById = Utils.createGuid('')
```

Comment:
I know this is tests, but it shouldn't be a `user`? and in the other cases in this file.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/1369/commits/03daaf722a0bff201708cf27b0434c8fbfef4f4c